utils.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 D-Wave Systems Inc.
+# Copyright 2024 D-Wave
```

Comment:
We _usually_ don't update this unless there are significant changes to the file.
```suggestion
# Copyright 2022 D-Wave
```